builtin/mainmenu/content/contentdb.lua

```diff
@@ -147,6 +147,11 @@ local function start_install(package, reason)
 			start_install(next.package, next.reason)
 		end
 
+		if package.type == "game" then
+			pkgmgr.update_gamelist()
+		else
+			pkgmgr.refresh_globals()
+		end
```

Comment:
`pkgmgr.install_dir` already does this

Reply:
For some reason, removing this causes ContentDB to no longer recognise install changes

Reply:
Ok, worked it out - it's because the `author` has not been set in the package's .conf at this point